lua/system/blueprints-props.lua

```diff
@@ -107,31 +107,35 @@ end
 --- - https://github.com/FAForever/fa/pull/4675
 ---@param prop PropBlueprint
 local function ProcessLOD(prop)
-
-    local sx = prop.SizeX or 1
-    local sy = prop.SizeY or 1
-    local sz = prop.SizeZ or 1
-
-    -- give more emphasis to the x / z value as that is easier to see in the average camera angle
-    local weighted = 0.40 * sx + 0.2 * sy + 0.4 * sz
-    if prop.ScriptClass == 'Tree' or prop.ScriptClass == 'TreeGroup' then
-        weighted = 10.0
-    end
-
-    -- https://www.desmos.com/calculator (0.9 * sqrt(100 * 500 * x))
-    local lod = 0.9 * MathSqrt(100 * 500 * weighted)
-
     if prop.Display and prop.Display.Mesh and prop.Display.Mesh.LODs then
-        local n = TableGetn(prop.Display.Mesh.LODs)
-        for k = 1, n do
-            local data = prop.Display.Mesh.LODs[k]
 
-            -- https://www.desmos.com/calculator (x * x)
-            local factor = (k / n) * (k / n)
-            local LODCutoff = factor * lod
+        local sx = prop.SizeX or 1
+        local sy = prop.SizeY or 1
+        local sz = prop.SizeZ or 1
+
+        -- give more emphasis to the x / z value as that is easier to see in the average camera angle
+        local weightedLodSize = MathSqrt(sx * sx + 0.5 * sy * sy + sz * sz)
+        local maxLod = 200 * weightedLodSize
+
+        if (prop.ScriptClass == 'Tree' or prop.ScriptClass == 'TreeGroup')
+          then
+            if TableGetn(prop.Display.Mesh.LODs) == 3 then
+                prop.Display.Mesh.LODs[1].LODCutoff = 40
+                prop.Display.Mesh.LODs[2].LODCutoff = 180
```

Comment:
```suggestion
                prop.Display.Mesh.LODs[2].LODCutoff = 165
```

This fixes the problem of the least detailed and intermediate LOD levels appearing at the same zoom level.

Reply:
Tested on a sensitivity of 100. We probably cannot fix this for all zoom levels, only specific ones.